src/tests/verification_mock/mod.rs

```diff
@@ -115,8 +115,8 @@ fn test_verification_without_mock_root() {
     verification_without_mock_root()
 }
 
-// Note: `WebPkiVerifier` (Linux) does not support IP address hosts, so these tests are disabled for Linux
-// Ref: https://docs.rs/rustls/0.20.6/src/rustls/verify.rs.html#322. Android does not currently support them either.
+// Note: Android does not currently support IP address hosts, so these tests are disabled for
+// Android.
```

Comment:
I've started working on this to address https://github.com/rustls/rustls-platform-verifier/issues/15 but I've reached the end of what I can accomplish iterating in CI (_my branch fails the `runMockTestSuite` step when running in the emulator without any helpful output_). 

I think I need to set up a proper Android dev env and getting Android Studio/The emulator/NDK/Gradle/Rust in NixOS is giving me a lot of grief.

Hoping to have a branch for this Soon.